app/src/main/java/com/example/pocketcloset/MainActivity.java

```diff
@@ -48,7 +50,10 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
                         fragment.setArguments(bundl);
                         break;
                     case R.id.action_outfits:
+                        Bundle bundle = new Bundle();
```

Comment:
Ideally you'd want this to be a static function on the Fragment, so as to not expose the internals. At the very least, you want the key "userToFilterBy" to be a global constant.